// jsonresume-theme-professional/build/ui/Education.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _Section = _interopRequireDefault(require("./Section"));
var _Experience = _interopRequireDefault(require("./Experience"));
var _jsxRuntime = require("react/jsx-runtime");
function _interopRequireDefault(e) { return e && e.__esModule ? e : { "default": e }; }
var Education = function Education(_ref) {
  var education = _ref.education;
  if (!education) {
    return null;
  }
  return /*#__PURE__*/(0, _jsxRuntime.jsx)("div", {
    children: /*#__PURE__*/(0, _jsxRuntime.jsx)(_Section["default"], {
      title: "Education",
      children: education.map(function (e, key) {
        var subTitle = e.area ? "".concat(e.studyType, " in ").concat(e.area) : e.studyType;
        if (e.score) {
          subTitle = "".concat(subTitle, " (").concat(e.score, ")");
        }
        return /*#__PURE__*/(0, _jsxRuntime.jsx)(_Experience["default"], {
          title: e.institution,
          subTitle: subTitle,
          startDate: e.startDate,
          endDate: e.endDate,
          highlights: e.courses
        }, key);
      })
    })
  });
};
var _default = exports["default"] = Education;
