// jsonresume-theme-professional/build/ui/Experience.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _styledComponents = _interopRequireDefault(require("styled-components"));
var _List = _interopRequireDefault(require("./List"));
var _DateRange = _interopRequireDefault(require("./DateRange"));
var _Date = _interopRequireDefault(require("./Date"));
var _jsxRuntime = require("react/jsx-runtime");
var _templateObject, _templateObject2, _templateObject3, _templateObject4, _templateObject5;
function _interopRequireDefault(e) { return e && e.__esModule ? e : { "default": e }; }
function _taggedTemplateLiteral(e, t) { return t || (t = e.slice(0)), Object.freeze(Object.defineProperties(e, { raw: { value: Object.freeze(t) } })); }
var Meta = _styledComponents["default"].div(_templateObject || (_templateObject = _taggedTemplateLiteral(["\n  display: flex;\n  justify-content: space-between;\n  margin-bottom: 2px;\n"])));
var Title = _styledComponents["default"].div(_templateObject2 || (_templateObject2 = _taggedTemplateLiteral(["\n  font-weight: 600;\n  font-size: 1.45rem;\n  margin-bottom: 3px;\n"])));
var SubTitle = _styledComponents["default"].div(_templateObject3 || (_templateObject3 = _taggedTemplateLiteral(["\n  font-style: italic;\n  font-size: 1.4rem;\n  margin-bottom: 3px;\n"])));
var Container = _styledComponents["default"].div(_templateObject4 || (_templateObject4 = _taggedTemplateLiteral(["\n  margin-bottom: 10px;\n"])));
var Summary = _styledComponents["default"].p(_templateObject5 || (_templateObject5 = _taggedTemplateLiteral(["\n  margin-bottom: 5px;\n"])));
var Experience = function Experience(_ref) {
  var title = _ref.title,
    date = _ref.date,
    startDate = _ref.startDate,
    endDate = _ref.endDate,
    subTitle = _ref.subTitle,
    summary = _ref.summary,
    highlights = _ref.highlights;
  return /*#__PURE__*/(0, _jsxRuntime.jsxs)(Container, {
    children: [/*#__PURE__*/(0, _jsxRuntime.jsxs)(Meta, {
      children: [/*#__PURE__*/(0, _jsxRuntime.jsx)(Title, {
        children: title
      }), /*#__PURE__*/(0, _jsxRuntime.jsx)("div", {
        className: "secondary",
        children: date ? /*#__PURE__*/(0, _jsxRuntime.jsx)(_Date["default"], {
          date: date
        }) : /*#__PURE__*/(0, _jsxRuntime.jsx)(_DateRange["default"], {
          startDate: startDate,
          endDate: endDate
        })
      })]
    }), subTitle && /*#__PURE__*/(0, _jsxRuntime.jsx)(SubTitle, {
      children: subTitle
    }), /*#__PURE__*/(0, _jsxRuntime.jsxs)("div", {
      className: "secondary",
      children: [summary && /*#__PURE__*/(0, _jsxRuntime.jsx)(Summary, {
        children: summary
      }), /*#__PURE__*/(0, _jsxRuntime.jsx)(_List["default"], {
        items: highlights
      })]
    })]
  });
};
var _default = exports["default"] = Experience;
