// jsonresume-theme-professional/build/ui/Resume.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _styledComponents = _interopRequireDefault(require("styled-components"));
var _Projects = _interopRequireDefault(require("./Projects"));
var _Hero = _interopRequireDefault(require("./Hero"));
var _Summary = _interopRequireDefault(require("./Summary"));
var _Education = _interopRequireDefault(require("./Education"));
var _Work = _interopRequireDefault(require("./Work"));
var _Certificates = _interopRequireDefault(require("./Certificates"));
var _Publications = _interopRequireDefault(require("./Publications"));
var _Awards = _interopRequireDefault(require("./Awards"));
var _Skills = _interopRequireDefault(require("./Skills"));
var _Interests = _interopRequireDefault(require("./Interests"));
var _Languages = _interopRequireDefault(require("./Languages"));
var _References = _interopRequireDefault(require("./References"));
var _jsxRuntime = require("react/jsx-runtime");
var _templateObject;
function _interopRequireDefault(e) { return e && e.__esModule ? e : { "default": e }; }
function _taggedTemplateLiteral(e, t) { return t || (t = e.slice(0)), Object.freeze(Object.defineProperties(e, { raw: { value: Object.freeze(t) } })); }
var Layout = _styledComponents["default"].div(_templateObject || (_templateObject = _taggedTemplateLiteral(["\n  max-width: 660px;\n  margin: 0 auto;\n  line-height: calc(1ex / 0.32);\n  margin-bottom: 40px;\n"])));
var Resume = function Resume(_ref) {
  var resume = _ref.resume;
  return /*#__PURE__*/(0, _jsxRuntime.jsxs)(Layout, {
    children: [/*#__PURE__*/(0, _jsxRuntime.jsx)(_Hero["default"], {
      basics: resume.basics
    }), /*#__PURE__*/(0, _jsxRuntime.jsx)(_Summary["default"], {
      basics: resume.basics
    }), /*#__PURE__*/(0, _jsxRuntime.jsx)(_Education["default"], {
      education: resume.education
    }), /*#__PURE__*/(0, _jsxRuntime.jsx)(_Work["default"], {
      work: resume.work
    }), /*#__PURE__*/(0, _jsxRuntime.jsx)(_Projects["default"], {
      projects: resume.projects
    }), /*#__PURE__*/(0, _jsxRuntime.jsx)(_Certificates["default"], {
      certificates: resume.certificates
    }), /*#__PURE__*/(0, _jsxRuntime.jsx)(_Publications["default"], {
      publications: resume.publications
    }), /*#__PURE__*/(0, _jsxRuntime.jsx)(_Awards["default"], {
      awards: resume.awards
    }), /*#__PURE__*/(0, _jsxRuntime.jsx)(_Languages["default"], {
      languages: resume.languages
    }), /*#__PURE__*/(0, _jsxRuntime.jsx)(_Skills["default"], {
      skills: resume.skills
    }), /*#__PURE__*/(0, _jsxRuntime.jsx)(_Interests["default"], {
      interests: resume.interests
    }), /*#__PURE__*/(0, _jsxRuntime.jsx)(_References["default"], {
      references: resume.references
    })]
  });
};
var _default = exports["default"] = Resume;
