// jsonresume-theme-professional/build/ui/Work.js
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports["default"] = void 0;
var _Section = _interopRequireDefault(require("./Section"));
var _Experience = _interopRequireDefault(require("./Experience"));
var _jsxRuntime = require("react/jsx-runtime");
function _interopRequireDefault(e) { return e && e.__esModule ? e : { "default": e }; }
var Work = function Work(_ref) {
  var work = _ref.work;
  if (!work) {
    return null;
  }
  return /*#__PURE__*/(0, _jsxRuntime.jsx)("div", {
    children: /*#__PURE__*/(0, _jsxRuntime.jsx)(_Section["default"], {
      title: "Experience",
      children: work.map(function (w, key) {
        return /*#__PURE__*/(0, _jsxRuntime.jsx)(_Experience["default"], {
          title: w.position,
          subTitle: w.name,
          startDate: w.startDate,
          endDate: w.endDate,
          summary: w.summary,
          highlights: w.highlights
        }, key);
      })
    })
  });
};
var _default = exports["default"] = Work;
